### PR TITLE
Drop the relationship between Packages and Users

### LINKED ISSUE #1997: The relationship between Packages and Users is unused
I noticed today while writing docblocks for the models that the Packages.committers relationship is always empty. To confirm this, I examined the database dump, and found that there really are no rows in the ```user_package_table``` through table:

```
--
-- Data for Name: user_package_table; Type: TABLE DATA; Schema: public; Owner: bodhi2
--

COPY user_package_table (user_id, package_id) FROM stdin;
\.
```

We should delete this relationship and table.

### PR DESCRIPTION
**Pull request by sebwoj**
Fixes #1997

Signed-off-by: Sebastian Wojciechowski <s.wojciechowski89@gmail.com>

### CODE REVIEW

File: bodhi/tests/server/test_models.py
Comment (bowlofeggs) [inline bodhi/tests/server/test_models.py:3226]: Heh, this test makes me think that we were supposed to do this. The problem is that nothing ever set the committers to anything on packages so this never actually happened. I guess we could fix Bodhi to actually set the committers and keep this table/code instead of deleting it. I'm not sure where would be the best place to do that though. Perhaps it would be one of the backend tasks when we create or edit an update?

What do you think? This is a feature that has obviously never worked, so I am OK with accepting this patch and dropping it. But I would also be OK with fixing it to what seems to be the intention here. Since you did the work, I'll let you decide ☺

Comment (sebwoj) [reply bodhi/tests/server/test_models.py:3226]: Hard to say. Is there any requirement for that in Bodhi?
Do you know about someone who mentioned that he didn't receive any notification?

Comment (bowlofeggs) [reply bodhi/tests/server/test_models.py:3226]: I don't know of any requirement to have this feature, and I've not heard from anyone saying they wanted e-mails that they weren't getting.

Comment (sebwoj) [reply bodhi/tests/server/test_models.py:3226]: If so, I think that we can remove this.

**Comment by bowlofeggs** [comment]
Cool, I am going to rebase this and merge.

**Comment by bowlofeggs** [comment]
I am applying this patch as well:

```
diff --git a/bodhi/server/migrations/versions/7ba286412ad4_drop_the_relationship_between_packages_.py b/bodhi/server/migrations/versions/7ba286412ad4_drop_the_relationship_between_packages_.py
index 633a8af3..cfa62a0b 100644
--- a/bodhi/server/migrations/versions/7ba286412ad4_drop_the_relationship_between_packages_.py
+++ b/bodhi/server/migrations/versions/7ba286412ad4_drop_the_relationship_between_packages_.py
@@ -19,7 +19,7 @@
 Drop the relationship between Packages and Users.
 
 Revision ID: 7ba286412ad4
-Revises: 190ba571c7d2
+Revises: 2fc96aa44a74
 Create Date: 2019-02-16 08:36:48.665825
 """
 from alembic import op
@@ -28,7 +28,7 @@ import sqlalchemy as sa
 
 # revision identifiers, used by Alembic.
 revision = '7ba286412ad4'
-down_revision = '190ba571c7d2'
+down_revision = '2fc96aa44a74'
 
 
 def upgrade():
```

**Review by bowlofeggs — APPROVED** [review]

**Comment by bowlofeggs** [comment]
The CI failures are due to the container registry being offline, which is due to the infrastructure mass reboot that is going on right now. We'll need to try again later.

**Comment by bowlofeggs** [comment]
This patch is planned for inclusion in the upcoming 4.0.0 release: #3221

**Comment by bowlofeggs** [comment]
Bodhi 4.0.0 beta is built and deployed to staging:

https://copr.fedorainfracloud.org/coprs/bowlofeggs/bodhi-pre-release
https://bodhi.stg.fedoraproject.org/

**Pull request merged**
